ruby.c

```diff
@@ -338,13 +340,15 @@ usage(const char *name, int help, int highlight, int columns)
         M("-W[level=2|:category]", "",             "Set warning flag ($-W):\n"
             "0 for silent; 1 for moderate; 2 for verbose."),
         M("-x[dirpath]",   "",			   "Execute Ruby code starting from a #!ruby line."),
-#if USE_YJIT
-        M("--jit",         "",                     "Enable JIT for the platform; same as " PLATFORM_JIT_OPTION "."),
+#if USE_YJIT || USE_ZJIT
+        M("--jit",         "",                     "Enable the default JIT for the build; same as " DEFAULT_JIT_OPTION "."),
 #endif
 #if USE_YJIT
         M("--yjit",        "",                     "Enable in-process JIT compiler."),
 #endif
-        M("--zjit",        "",                     "Enable in-process JIT compiler."),
+#if USE_ZJIT
```

Comment:
elif here, as above?

Reply:
It's an deliberate choice. There can be only one default option, hence `#elif` for it, but non-default options can co-exist.